walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java

```diff
@@ -30,4 +30,24 @@ public void testGreeting() {
 
     Assert.assertEquals("Hello Ada", greeting);
   }
+
+  @Test
+  public void testGreetingTrimsWhitespace() {
+    Greeter greeter = new Greeter();
```

Comment:
Nice job meeting the "arrange, act, test" model of writing Java unit tests at Google! 

In case you're interested... the arrange step is for setting up details of the external world specific to the situation under test (e.g. in this case, initializing the `Greeter`). 

The act step is where you actually make calls to the class/object under test .

The assert step is the place to make assertions on the return values collected.

We usually leave a blank line in between the arrange, act, and test steps - which you've already done!

Reply:
Thank you, and that's very interesting!